rerun_py/rerun_sdk/rerun/__main__.py

```diff
@@ -1,15 +1,17 @@
 """See `python3 -m rerun --help`."""
 from __future__ import annotations
 
-import sys
+import rerun_bindings as bindings
 
-from rerun import bindings, unregister_shutdown
+from rerun import unregister_shutdown
 
 
 def main() -> None:
     # We don't need to call shutdown in this case. Rust should be handling everything
+    # TODO(ab): we only do that because __init__.py was loaded and register_shutdown() was called.
+    # Ideally, nothing should be loaded at all but `rerun_bindings` when we run `python3 -m rerun`.
     unregister_shutdown()
-    exit(bindings.main(sys.argv))
+    exit(bindings.main())
```

Comment:
This confuses me.

"We don't need to call shutdown … we only do that because …" - wait, so do we need to call it or not? And is calling `exit` = "call shutdown"?

Reply:
Too many negations :) I've clarified the comment.